packages/api/prisma/schema.prisma

```diff
@@ -24,6 +24,7 @@ model Round {
   isSaturated        Boolean        @default(false)
   roundSummary       RoundSummary?
   projects           Project[]
+  votes              Vote[]
```

Comment:
Shouldn't this be optional ? Not every project would have a vote right ?

Reply:
The array is optional! There can be no elements!